packages/npm/src/index.ts

```diff
@@ -1 +1,10 @@
 export * from './utils';
+
+// generated by Theo
```

Comment:
This comment gives me the impression that these lines were generated, rather than the files they import.

Perhaps because they match the first line of actually generated files.

I wonder, maybe `// Theo generated files` like you did in `.gitignore`?

Reply:
I don't mind renaming, but yes I want to keep the comment.

The reason is that such do not exist if you do nor run a "generate" (or "build") script, so just to get the attention on why that is.

Reply:
Yea I'd like to keep to comment too.

Just different wording to make it clear that it's the files being imported that are generated, not the lines under it.